package.json

```diff
@@ -11,7 +11,7 @@
     "dashboarding"
   ],
   "private": true,
-  "version": "3.0.0",
+  "version": "2.8.0",
```

Comment:
Please revert

Reply:
will revert to 3.0.0 for PR purpose.

But the opensearch 3.0.0 snapshot has some issues that are causing some visualizations fail to load on dashboard, and also unable to create visualizations directly from dashboard.